app/components/selector_component.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class SelectorComponent < ViewComponentReflex::Component
+  def initialize(title:, selected:, items:, data: {})
+    super
+    @title = title
+    @items = items.map do |item|
+      {
+        label: item[:label],
+        value: item[:value],
+        selected: selected.include?(item[:value])
+      }
+    end
+    @selected = selected
+    @data = data
+  end
+end
```

Comment:
This logic looks exactly like the [select helper](https://apidock.com/rails/ActionView/Helpers/FormOptionsHelper/select) in Rails. Can we re-use native browser elements and style them? Like in the reports?